From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Wed, 3 Jul 2013 21:00:45 +0100
Subject: AUXDIR_NAME sans arch

---
 src/configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/configure.ac b/src/configure.ac
index 09237da..f07ce71 100644
--- a/src/configure.ac
+++ b/src/configure.ac
@@ -102,7 +102,7 @@ svm1)
     INSTALL_LIARC_BUNDLES=
     ;;
 esac
-AUXDIR_NAME=mit-scheme-${mit_scheme_native_code}
+AUXDIR_NAME=mit-scheme
 AUXDIR='$(libdir)'/${AUXDIR_NAME}
 
 # Please keep the following messages synchronized with the messages in
